content/sensu-go/6.1/observability-pipeline/observe-events/events.md

```diff
@@ -120,7 +120,32 @@ Sensu events contain:
 ## Use event data
 
 Observability data in events is a powerful tool for automating monitoring workflows.
-For example, you can [reduce alert fatigue][23] by filtering events based on the event `occurrences` attribute.
+For example, you can use the [`state` attribute][36] to provide handlers with more information about changes in check status and [reduce alert fatigue][23] by filtering events based on the event `occurrences` attribute.
+
+### State attribute
+
+The `state` event attribute adds meaning to the check status:
+
+- `passing` means the check status is `0` (OK).
+- `failing` means the check status changed from 0 to non-zero (WARNING or CRITICAL).
+- `flapping` means a rapid change in check result status (determined based on the [low and high flap thresholds][37] set in the check attributes).
+
+Flapping indicates problems with an entity, provided your low and high flap threshold settings are properly configured.
+Tuning your flap threshold configuration based on Sensu's [flap detection algorithm][39] allows you to take action based on `flapping` state check results.
+
+#### Flap detection algorithm
+
+Sensu uses the same flap detection algorithm as [Nagios][38].
+Every time you run a check, Sensu records whether the `status` value changed since the previous check.
+Sensu stores the last 21 `status` values and uses them to calculate an entity's percent state change.
```

Comment:
```suggestion
Sensu stores the last 21 `status` values and uses them to calculate the percent state change for the entity/check pair.
```

Reply:
The algorithm is applied to entity/check pairs. As written, some of this seems to be describing percent state change across all check results for an entity.